explorer/Dockerfile

```diff
@@ -2,13 +2,27 @@ FROM node:10.16.3 as builder
 
 RUN apt-get update && apt-get install -y libudev-dev libusb-dev libusb-1.0-0
 
-RUN mkdir -p /usr/src/app
+RUN mkdir -p /usr/src/app/
 WORKDIR /usr/src/app
 ENV PATH /usr/src/app/node_modules/.bin:$PATH
 
-ADD . .
-
-RUN yarn install
+COPY yarn.lock package.json .yarnrc .yarn/ ./
+COPY .yarn .yarn
+COPY explorer/package.json ./explorer/
+COPY explorer/client/package.json ./explorer/client/
+COPY tools/package.json ./tools/
+COPY tools/json-api-client/package.json ./tools/json-api-client/
+COPY tools/local-storage/package.json ./tools/local-storage/
+COPY tools/redux/package.json ./tools/redux/
+COPY tools/ts-helpers/package.json ./tools/ts-helpers/
+COPY styleguide/package.json ./styleguide/
+RUN yarn install --frozen-lockfile
+
+ADD tsconfig.*.json babel.config.js ./
```

Comment:
Any reason why we're using `ADD` over `COPY` here and below? I think `COPY` should work fine

Reply:
We can add these files to line 9 FWIW, they're barely ever changed